catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -115,7 +123,8 @@ public String getWkt() throws CswException {
         // started
         reader.moveUp();
 
-        LOGGER.debug("Returning WKT in LON/LAT coord order: {}.", wkt);
+        wkt = convertToEPSG4326(wkt, crs);
```

Comment:
There is no guarantee of the ordering for EPSG:4326 coming in, so we cannot handle this internally.  Thus, we should still allow the Admin to specify the ordering in the CSW Source config.  Here's a good read : http://docs.geotools.org/latest/userguide/library/referencing/order.html